lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -541,13 +671,13 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
     IO.iodata_to_binary(Enum.reverse(buffer))
   end
 
-  defp text_to_acc(buffer, acc, line, column, context)
+  defp text_to_acc(buffer, acc, line_offset, column, context)
 
-  defp text_to_acc([], acc, _line, _column, _context),
+  defp text_to_acc([], acc, _line_offset, _column, _context),
     do: acc
 
-  defp text_to_acc(buffer, acc, line, column, context) do
-    meta = %{line_end: line, column_end: column}
+  defp text_to_acc(buffer, acc, {line, offset}, column, context) do
+    meta = %{line_end: line, column_end: column, offset: offset}
```

Comment:
I don't think we want to include the offset in text nodes, I think it is rather on opening and closing tags.